src/components/Sections/SectionProgress.tsx

```diff
@@ -1,41 +1,83 @@
 import React from 'react'
-import { Icon, Label, Progress } from 'semantic-ui-react'
-import {
-  ApplicationProgress,
-  ReviewAndConsolidationProgress,
-  ReviewProgress,
-  SectionState,
-} from '../../utils/types'
+import { Label, Progress } from 'semantic-ui-react'
+import { ApplicationProgress, ConsolidationProgress, ReviewProgress } from '../../utils/types'
 import strings from '../../utils/constants'
 
-const ReviewStatusOrProgress: React.FC<SectionState> = ({
-  reviewProgress,
-  reviewAction,
-  reviewAndConsolidationProgress,
-}) => {
-  if (reviewAction?.isAssignedToCurrentUser && reviewProgress) {
-    return reviewAction.isReviewable ? (
-      <ReviewSectionProgressBar
-        reviewProgress={reviewProgress}
-        reviewAndConsolidationProgress={reviewAndConsolidationProgress}
-      />
-    ) : (
-      <Label
-        icon={<Icon name="circle" size="mini" color="blue" />}
-        content={strings.LABEL_ASSIGNED_TO_YOU}
+interface SectionProgressBarProps {
+  consolidationProgress?: ConsolidationProgress
+  reviewProgress?: ReviewProgress
+}
+
+/**
+ * Application
+ */
+
+const getApplicationProgressTitle = ({ completed, valid }: ApplicationProgress) => {
+  if (!valid) return strings.LABEL_SECTION_PROBLEM
+  else if (completed) return strings.LABEL_SECTION_COMPLETED
+  return null
+}
+
+const ApplicationProgressBar: React.FC<ApplicationProgress> = (applicationProgress) => {
+  const { doneRequired, doneNonRequired, totalSum, valid } = applicationProgress
+  const totalDone = doneRequired + doneNonRequired
+  const progressLabel = getApplicationProgressTitle(applicationProgress)
+  return totalDone > 0 && totalSum > 0 ? (
+    <div className="progress-box">
+      {progressLabel && <Label size="tiny" className="simple-label" content={progressLabel} />}
+      <Progress
+        className="progress"
+        percent={(100 * totalDone) / totalSum}
+        size="tiny"
+        success={valid}
+        error={!valid}
       />
-    )
-  }
-  return <Label className="simple-label" content={strings.LABEL_ASSIGNED_TO_OTHER} />
+    </div>
+  ) : null
 }
 
-const getReviewProgressDefaults = ({
-  reviewProgress,
-  reviewAndConsolidationProgress,
-}: SectionProgressBarProps) => ({
+/**
+ * Consolidation
+ */
+
+const getConsolidationProgressDefaults = ({ consolidationProgress }: SectionProgressBarProps) => ({
+  doneAgreeNonConform: consolidationProgress?.doneAgreeNonConform || 0,
+  doneAgreeConform: consolidationProgress?.doneAgreeConform || 0,
+  doneDisagree: consolidationProgress?.doneDisagree || 0,
+  totalReviewable: consolidationProgress?.totalReviewable || 0,
+  totalPendingReview: consolidationProgress?.totalPendingReview || 0,
+})
+
+const ConsolidationSectionProgressBar: React.FC<SectionProgressBarProps> = (props) => {
+  const {
+    doneAgreeNonConform,
+    doneAgreeConform,
+    doneDisagree,
+    totalReviewable,
+    totalPendingReview,
+  } = getConsolidationProgressDefaults(props)
+  return (
+    <Progress
```

Comment:
should there be a label (disagree, agree non-conform, agree conform) or something like that ?

Reply:
Yes, it would be nice. I guess I wasn't sure what to display exactly. Any suggestions? I can try something here quickly now : )

Reply:
Done change here 4639a35, but haven't tested yet. Will do that on my current unPRd of #275 & #276 - Should be on a PR today : )